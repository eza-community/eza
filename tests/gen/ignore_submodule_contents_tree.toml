bin.name = "eza"
args = "tests/test_dir/git/with_submodule --tree --ignore-submodule-contents"
