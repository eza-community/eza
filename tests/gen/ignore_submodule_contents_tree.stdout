tests/test_dir/git/with_submodule
├── 001
├── file_000
├── file_001
├── file_002
├── file_003
├── file_004
├── file_005
├── file_006
├── file_007
├── file_008
└── file_009
